spec/graphql/static_validation/rules/argument_literals_are_compatible_spec.rb

```diff
@@ -65,4 +65,126 @@
     }
     assert_includes(errors, fragment_error)
   end
+
+  describe "null value" do
```

Comment:
If the type is `[Int]!` it cannot be `null`, but it could be `[null]`.

As far as I understand, non-null types cannot be `null`.